src/app/shared/pages/collateral/collateral-page/collateral-page.component.html

```diff
@@ -115,6 +116,13 @@ <h2>Sikkerhet i bolig</h2>
           Nei
         </mat-radio-button>
       </mat-radio-group>
+      <mat-error *ngIf="showError()">*Velg om du har medlåntaker</mat-error>
+      </form>
+    </div>
+    <div class='navigation-button-container'>
```

Comment:
Bruk section i stedet for div og fjern klasse

Reply:
Dette er bare kopiert fra det du hadde gjort i broker, men fikser

Reply:
Ser den! Har endret i broker på en annen PR